src/exponential.jl

```diff
@@ -13,10 +13,40 @@ Reverse function for `exp`.
 function exp_rev(y::Interval, x::Interval)
```

Comment:
You could write these three functions using metaprogramming since they are so similar.
(But probably it's not worth it.)